### PR TITLE
Commerce workaround for broken qt5.10

### PR DESCRIPTION
**Pull request by howard-stearns**
https://highfidelity.fogbugz.com/f/cases/13738/Loading-graphic-displays-infinitely-on-Marketplace

### CODE REVIEW

**Comment by howard-stearns** [comment]
Test Plan:
Open MARKETPLACE **in staging**. (Commerce only works in staging right now, separately from this PR.)
Ignore the formatting. That's a separate ops issue.
Change the dropdown to any other category.
Make sure that any spinners that appear do go away in a timely way.
Also check other HTML apps, such as HELP and SNAPS.

File: scripts/system/help.js [hunk 1]
```diff
@@ -40,7 +40,7 @@
     }
 
     function onScreenChanged(type, url) {
-        onHelpScreen = type === "Web" && url.startsWith(HELP_URL);
+        onHelpScreen = type === "Web" && (url.indexOf(HELP_URL) === 0);
```
Comment (howard-stearns) [inline scripts/system/help.js]: This is really separate, been has been broken for at least a month.
There is no startsWith in QTs javascript, and so spews nasty stuff in the log.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8456/HighFidelity-Beta-PR12772-38f26a26476c9fe18ee735e9410df947926d5fbe.dmg

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Currently Testing

**Pull request merged**